clang/include/indexstore/indexstore.h

```diff
@@ -129,10 +129,24 @@ INDEXSTORE_PUBLIC unsigned
 indexstore_format_version(void);
 
 typedef void *indexstore_t;
+typedef void *indexstore_creation_options_t;
+
+INDEXSTORE_PUBLIC indexstore_creation_options_t
+indexstore_creation_options_create();
+
+INDEXSTORE_PUBLIC void
+indexstore_creation_options_dispose(indexstore_creation_options_t);
+
+INDEXSTORE_PUBLIC void indexstore_creation_options_add_prefix_mapping(
+    indexstore_creation_options_t options, const char *path_prefix, const char *remapped_path_prefix);
 
 INDEXSTORE_PUBLIC indexstore_t
 indexstore_store_create(const char *store_path, indexstore_error_t *error);
 
+INDEXSTORE_PUBLIC indexstore_t
+indexstore_store_create_with_options(const char *store_path, indexstore_creation_options_t options,
```

Comment:
Could you add a doc-comment to clarify that `options` can be `NULL`?

Reply:
Done